static/app/views/issueDetails/groupFeatureFlags/flagDrawerContent.tsx

```diff
@@ -64,6 +66,25 @@ export default function FlagDrawerContent({
     return searchedTags;
   }, [data, search, tagValues]);
 
+  // Suspect flag scoring. This a rudimentary INTERNAL-ONLY display for testing our scoring algorithms.
+  const organization = useOrganization();
+  const showScores = organization.features.includes(
```

Comment:
would it be possible to add some kind of state/toggle here to hide the scores? seems pretty noisy and would love the option to not see it if i want to focus on the flags